### PR TITLE
Formatting sql

### PR DESCRIPTION
**Pull request by MikeyBronowski**
Added 'sql' to blocks with T-SQL code
Removed pre, code blocks
Fixed minor typos.

### CODE REVIEW

**Comment by PRMerger12** [comment]
@MikeyBronowski : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/sql-server/sql-server-2012-release-notes.md [hunk 1]
```diff
@@ -99,8 +101,9 @@ You can install SQL Server on Windows Server 2008 R2 Server Core SP1, with the f
   
 If these tasks are not completed, you will see the following error message when you try to create a semantic index.  
   
-<pre>Msg 41209, Level 16, State 3, Line 1  
-A semantic language statistics database is not registered. Full-text indexes using 'STATISTICAL_SEMANTICS' cannot be created or populated.</pre>  
+```Msg 41209, Level 16, State 3, Line 1
```
Comment (rothja) [inline docs/sql-server/sql-server-2012-release-notes.md:104]: We have to stick to regular code block values here or just leave it blank. Thanks! Please move the Msg line into the body of the code block. Thank you!

File: docs/sql-server/sql-server-2012-release-notes.md [hunk 2]
```diff
@@ -56,13 +56,15 @@ Before installing [!INCLUDE[ssCurrent](../includes/sscurrent-md.md)], consider t
...
-Service 'SQLBrowser' start request failed. Click 'Retry' to retry the failed action, or click 'Cancel' to cancel this action and continue setup.</pre>  
+```The following error has occurred:  
+Service 'SQLBrowser' start request failed. Click 'Retry' to retry the failed action, or click 'Cancel' to cancel this action and continue setup.
+```
   
 or  
   
-<pre>The following error has occurred:  
-SQL Server Browser configuration for feature 'SQL_Browser_Redist_SqlBrowser_Cpu32' was cancelled by user after a previous installation failure. The last attempted step: Starting the SQL Server Browser service 'SQLBrowser', and waiting for up to '900' seconds for the process to complete.</pre>  
+```The following error has occurred:
```
Comment (rothja) [inline docs/sql-server/sql-server-2012-release-notes.md:65]: You have to move the "The following error has occurred:" into the code block. Thanks!

File: docs/integration-services/lesson-1-create-a-project-and-basic-package-with-ssis.md [hunk 3]
```diff
@@ -53,7 +54,7 @@ Here is an example of the source data in the SampleCurrencyData.txt file:
 1.00070049USD9/9/05 0:001.001502253  
 1.00070049USD9/10/05 0:000.99990001  
 1.00020004USD9/11/05 0:001.001101211  
-1.00020004USD9/12/05 0:000.99970009</pre>  
+1.00020004USD9/12/05 0:000.99970009```
```
Comment (rothja) [inline docs/integration-services/lesson-1-create-a-project-and-basic-package-with-ssis.md:57]: Please put the tick marks on its own line after this code block.

File: docs/2014/database-engine/implementing-lob-columns-in-a-memory-optimized-table.md [hunk 4]
```diff
@@ -69,7 +69,7 @@ where BlobId = @BlobId
 order by SegmentationId  
   
 select @Blob  
-go</code></pre>  
+go```
```
Comment (rothja) [inline docs/2014/database-engine/implementing-lob-columns-in-a-memory-optimized-table.md:72]: Please put closing tick marks on its own line. Thank you!

**Comment by PRMerger8** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/integration-services/lesson-1-create-a-project-and-basic-package-with-ssis.md (hunk 3 above)
Comment (ktoliver) [reply docs/integration-services/lesson-1-create-a-project-and-basic-package-with-ssis.md:57]: I can apply these fixes . . .

**Comment by PRMerger9** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/sql-server/sql-server-2012-release-notes.md (hunk 1 above)
Comment (ktoliver) [reply docs/sql-server/sql-server-2012-release-notes.md]: I made this fix.

File: docs/2014/database-engine/implementing-lob-columns-in-a-memory-optimized-table.md (hunk 4 above)
Comment (ktoliver) [reply docs/2014/database-engine/implementing-lob-columns-in-a-memory-optimized-table.md]: I made this fix.

File: docs/sql-server/sql-server-2012-release-notes.md (hunk 2 above)
Comment (ktoliver) [reply docs/sql-server/sql-server-2012-release-notes.md]: Done.

**Comment by rothja** [comment]
Thank you @ktoliver  and @MikeyBronowski . #sign-off.

**Pull request merged**